### ISSUE TITLE
TypeInitializationException.

### ISSUE THREAD

**Issue by prozolic** [xin9le/FastEnum#58]
When the following code is executed, `TypeInitializationException` is thrown. 
 The reason is that 'A' and 'a' are included in the Enum value, is this a specification?
 
```csharp
var str = Alphabet.A.FastToString();

public enum Alphabet
{
    A,
    a
}
```
```
System.TypeInitializationException: 'The type initializer for 'FastEnumUtility.Internals.EnumInfo`1' threw an exception.'
```

**Comment by xin9le** [comment]
No, this is not a specification issue. It is a bug due to missing test cases, which will be fixed in the near future.

**Comment by xin9le** [comment]
Thank you for your feedback regarding edge cases. We have released version [2.0.4](https://github.com/xin9le/FastEnum/releases/tag/v2.0.4), and I kindly invite you to utilize the updated release.
